amethyst_network/src/network_socket.rs

```diff
@@ -28,6 +30,16 @@ struct RawEvent {
     pub source: SocketAddr,
 }
 
+impl From<Packet> for RawEvent {
```

Comment:
I would also maybe implement `From<RawEvent>` for `Packet`.

Reply:
Not sure if we can implement something on a type we don't own and is from another crate.

Reply:
Fixed it thanks!